packages/mrl/src/mrl.interfaces.ts

```diff
@@ -41,3 +45,10 @@ export interface ChainTransferData {
   fee: AssetAmount;
   min: AssetAmount;
 }
+
+// TODO this is just for Wormhole
```

Comment:
not sure what this comment means

Reply:
This means that the RedeemData only make sense if we consider that the provider is Wormhole. If you look at the properties (vaa, tokenTransfer), its types are from the wormhole-sdk. So if in the future we add another provider / bridge, this data will not make sense. 
Not sure how to fix it for the moment